internal/display/users.go

```diff
@@ -0,0 +1,117 @@
+package display
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/auth0/auth0-cli/internal/ansi"
+	"gopkg.in/auth0.v5"
+	"gopkg.in/auth0.v5/management"
+)
+
+type userView struct {
+	UserID     string
+	Email      string
+	Connection string
```

Comment:
Please add a field for username, and display it only when there is a value.

Reply:
This will require using another struct for the key/value list, because the table view _should not_ include the username.
So, all in all: 
- In the table view struct: the id, email and connection, in that order.
- In the key/value struct: the id, connection, email and username, in that order. The key/value list should only display the username field when there is a username value.

Reply:
i'm not sure i quite understood this. you can check the current implementation of my recent commit. I'm using one struct, `userView` and added the username field to it. key values will display username if there is one and vice versa. you can review `../display/users.go` file

Reply:
Yes, I misunderstood. Sorry.